spec/lib/tasks/withdraw_and_redirect_section_rake_spec.rb

```diff
@@ -25,6 +25,7 @@
         "1234",
         type: "redirect",
         alternative_path: "/guidance/parent_path",
+        discard_drafts: true,
```

Comment:
It doesn't feel quite right that we can just change a line in the assertion here and not change any inputs or have to create a new test. Probably reflects more on there being a missing test case for the other side of the condition before though